R/expand_model_out_grid.R

```diff
@@ -529,3 +547,47 @@ validate_output_types <- function(output_types, config_tasks, round_id,
   }
   valid_output_types
 }
+
+validate_derived_task_ids <- function(derived_task_ids, config_tasks, round_id) {
+  checkmate::assert_character(derived_task_ids, null.ok = TRUE)
+  if (is.null(derived_task_ids)) {
+  }
```

Comment:
Empty code block; remove or was there content to be added here?

Reply:
DOH!! Fixed in c6c54237d22cdffc18d5eca811fd7331cd6c698b 🙈